saleor/graphql/views.py

```diff
@@ -120,7 +120,10 @@ def render_playground(self, request):
         return render(
             request,
             "graphql/playground.html",
-            {"api_url": request.build_absolute_uri(str(API_PATH))},
+            {
+                "api_url": request.build_absolute_uri(str(API_PATH)),
+                "plugins_url": request.build_absolute_uri("/plugins/"),
```

Comment:
Looks good but shouldn't we have this URL defined in `urls.py` and use the `reverse` function here? I'm not sure as there is no view `/plugins/`; there is the `plugins/(?P<plugin_id>[.0-9A-Za-z_\-]+)/` named URL but it requires the ID.

Reply:
reverse won't work because it's not defined in url.py, but build_absolute_uri works because it just takes a path and returns it relative to the current host

P.S: This is tested

Reply:
👍